_posts/2015-10-07-foreman-community-newsletter-october.md

```diff
@@ -0,0 +1,65 @@
+---
+layout: post
+title: Foreman Community Newsletter - October 2015
+date: '2015-10-07T15:28:00.001+03:00'
+author: Ori Rabin
+tags:
+- katello
+- openscap
+- foreman
+- discovery
+- community
+- hammer
+- chef
+- demo
+- newsletter
+modified_time: '2015-10-07T15:28:10.512+03:00'
+blogger_id: tag:blogger.com,1999:blog-6789674575954398874.post-7066927474050736906
+blogger_orig_url: http://blog.theforeman.org/2015/10/foreman-community-newsletter-october.html
+excerpt: |
+  Foreman 1.9.2 was released at the end of September, Katello 2.3.1 is out, new
+  plugins, events, and a new community manager. So much is happening this month!
+---
+
+### Foreman 1.9.2 released
+
+Foreman 1.9.2 was released at the end of September fixing a regression for
+smart class parameters introduced in 1.9.1 and more bugs. More details can be
+found in the
+[announcement](https://groups.google.com/forum/#!topic/foreman-announce/TQqnzHmBJOY).
+
+### **Foreman 1.8.4 released**
```

Comment:
I don't think I added that - I did *remove* bold from the 1.9.2 header while attempting to sort out the excerpts, which is possibly why they now look inconsistent. I'll put it back.